### PR TITLE
libraries/nyxt-asdf: Report error when non-interactive.

### PR DESCRIPTION
**Pull request by Ambrevar**
# Description

The CI failed to run the tests since https://github.com/atlas-engineer/nyxt/pull/2553, and worse, it failed to report the failure :(  So effectively nothing got reported at all...

This should fix it.

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.

- [x] I have pulled from master before submitting this PR
- [x] There are no merge conflicts.
- [x] I've added the new dependencies as:
  - [ ] ASDF dependencies,
  - [ ] Git submodules,
    ```sh
	cd /path/to/nyxt/checkout
    git submodule add https://gitlab.common-lisp.net/nyxt/py-configparser _build/py-configparser
    ```
  - [ ] and Guix dependencies.
- [x] My code follows the style guidelines for Common Lisp code. See:
  - [Norvig & Pitman's Tutorial on Good Lisp Programming Style (PDF)](https://www.cs.umd.edu/~nau/cmsc421/norvig-lisp-style.pdf)
  - [Google Common Lisp Style Guide](https://google.github.io/styleguide/lispguide.xml)
- [x] I have performed a self-review of my own code.
- [ ] My code has been reviewed by at least one peer.  (The peer review to approve a PR counts.  The reviewer must download and test the code.)
- [x] Documentation:
  - [ ] All my code has docstrings and `:documentation`s written in the aforementioned style.  (It's OK to skip the docstring for really trivial parts.)
  - [ ] I have updated the existing documentation to match my changes.
  - [ ] I have commented my code in hard-to-understand areas.
  - [ ] I have updated the `changelog.lisp` with my changes if it's anything user-facing (new features, important bug fix, compatibility breakage).
  - [ ] I have added a `migration.lisp` entry for all compatibility-breaking changes.
- [ ] Compilation and tests:
  - [ ] My changes generate no new warnings.
  - [ ] I have added tests that prove my fix is effective or that my feature works.  (If possible.)
  - [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by aartaka** [comment]
:(

The fix looks alright, yes!

**Comment by Ambrevar** [comment]
The CI is properly failing! Hooray :)

Let's fix it now.

File: libraries/prompter/prompter.lisp [hunk 1]
```diff
@@ -466,8 +471,14 @@ After timeout has elapsed for one source, return nil."
 (export-always 'make)
 (define-function make
     (append '(&rest args)
-            `(&key ,@(public-initargs 'prompter)))
-  "Return prompter object."
+            `(&key sources ,@(public-initargs 'prompter)))
+  "Return prompter object.
+The arguments are the initargs of the `prompter' class.
+
+As a special case, the `:source' keyword argument not only accepts `source'
```
Comment (aartaka) [inline libraries/prompter/prompter.lisp:478]: You mean `:sources`?

Comment (Ambrevar) [reply libraries/prompter/prompter.lisp:478]: Yup!

**Comment by Ambrevar** [comment]
Merged with 9116e5e58898a6c552b88aab2a07b71a7414813f.

**Pull request merged**